aws/application/parameters/development-maat-cd-api-pipeline.json

```diff
@@ -18,6 +18,7 @@
     "pLaaProsecutionConcludedQueue": "crime-apps-dev-prosecution-concluded-queue",
     "pCdaOauthUrl": "https://laa-court-data-adaptor-dev.apps.live-1.cloud-platform.service.justice.gov.uk/oauth/token",
     "pCdaLaaStatusUrl": "https://laa-court-data-adaptor-dev.apps.live-1.cloud-platform.service.justice.gov.uk/api/internal/v1/representation_orders",
+    "pCdaHearingUrl": "https://laa-court-data-adaptor-dev.apps.live-1.cloud-platform.service.justice.gov.uk/api/internal/v1/hearings/{hearingId}",
```

Comment:
Can we pass the sitting day as one of its parameters to make sure multi-day hearing scenarios are addressed too?

Reply:
As per our discussion the get hearing endpoint should return the aggregated view of the hearing so even for multi-day hearings we _should_ be ok.